DuckDuckGo/UserBehaviorMonitor.swift

```diff
@@ -64,32 +71,46 @@ final class UserBehaviorMonitor {
         set { store.didRefreshTimestamp = newValue }
     }
 
-    var didBurnTimestamp: Date? {
-        get { store.didBurnTimestamp }
-        set { store.didBurnTimestamp = newValue }
+    var didDoubleRefreshTimestamp: Date? {
+        get { store.didDoubleRefreshTimestamp }
+        set { store.didDoubleRefreshTimestamp = newValue }
+    }
+
+    var didRefreshCounter: Int {
+        get { store.didRefreshCounter }
+        set { store.didRefreshCounter = newValue }
     }
 
     func handleAction(_ action: Action, date: Date = Date()) {
         switch action {
         case .refresh:
-            fireEventIfActionOccurredRecently(since: didRefreshTimestamp, eventToFire: .reloadTwice, within: 10.0)
+            fireEventIfActionOccurredRecently(within: 12.0, since: didRefreshTimestamp, eventToFire: .reloadTwiceWithin12Seconds)
+            fireEventIfActionOccurredRecently(within: 24.0, since: didRefreshTimestamp, eventToFire: .reloadTwiceWithin24Seconds)
             didRefreshTimestamp = date
-        case .burn:
-            fireEventIfActionOccurredRecently(since: didRefreshTimestamp, eventToFire: .reloadAndFireButton)
-            didBurnTimestamp = date
+            
+            if didRefreshCounter == 0 {
+                didDoubleRefreshTimestamp = date
+            }
+            didRefreshCounter += 1
+            if didRefreshCounter == 3 {
```

Comment:
`> 2` is a safer option incase that counter gets incremented unexpectedly, to avoid this getting in a weird state that never gets called